### PR TITLE
fix(mysql): improve random(with LCG) password for mysql user

### PR DESCRIPTION
**Pull request by imzhengfei**
fix: #511

### CODE REVIEW

**Comment by imzhengfei** [comment]
Use Linear Congruential Generator to generate a random number between 0 and 1, in stead of Math.random().

File: extensions/mysql/index.js [hunk 1]
```diff
@@ -77,7 +76,74 @@ class MySQLExtension extends cli.Extension {
     }
 
     createUser(ctx, dbconfig) {
-        const randomPassword = crypto.randomBytes(10).toString('hex');
+        // Generate random MySQL password
+        const generateRandomPassword = () => {
+            /**
+             * LCG Random
+             * @example cosnt random = (lcgRandom())
```
Comment (vikaspotluri123) [inline extensions/mysql/index.js]: Was this supposed to say `const` instead of `cosnt`?

Comment (imzhengfei) [reply extensions/mysql/index.js]: Oh! Sorry! Just a spelling error.

**Comment by imzhengfei** [comment]
fixed a spelling error

**Comment by imzhengfei** [comment]
@acburdine I am not sure, do you mean MySQLExtension class ?

**Comment by acburdine** [comment]
@imzhengfei my apologies, yes that's what I meant.

However, as I commented [here](https://github.com/TryGhost/Ghost-CLI/pull/534#issuecomment-344140943), there's another issue in that the initial `create user` command is what actually is triggering the insecure password error - so we need to find a way around that as well.

**Comment by acburdine** [comment]
Closing in favor of #546 -> ultimately it required other changes than this. @gompa's approach in #534 was a bit better than this I think too - it was less code to maintain + the library used in that PR uses crypto.randomBytes rather than something based off of a technically predictable seed.

**Pull request closed without merging**